src/main/include/Talon.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <units/angle.h>
+#include <units/length.h>
+#include "Constants.h"
+
+namespace t34
+{
+  namespace Talon
+  {
+    constexpr auto SRX_RESOLUTION         { 4096.0 / 1_tr }; //4096 units per rotation
+    constexpr auto _775PRO_SHAFT_CIRCUMFERENCE{ (2 * (M_PI * (0.197_in / 2))) / 1_tr };
```

Comment:
You're multiplying by 2 _and_ dividing by 2.

I take it that `0.197_in` is the shaft diameter. As both $2 \pi r$ and $\pi d$ are well-known expressions of a circumference, I think it would be acceptable simplify it:

```suggestion
    constexpr auto _775PRO_SHAFT_CIRCUMFERENCE{ (M_PI * 0.197_in) / 1_tr };
```